daphne/src/lib.rs

```diff
@@ -378,7 +372,10 @@ impl DapGlobalConfig {
         let kem_ids = self.supported_hpke_kems.clone();
         kem_ids.into_iter().enumerate().map(move |(i, kem_id)| {
             let (config_id, _overflowed) = first_config_id.overflowing_add(i as u8);
-            HpkeReceiverConfig::gen(config_id, kem_id)
+            match HpkeReceiverConfig::gen(config_id, kem_id) {
+                Ok(cfg) => cfg,
+                Err(e) => panic!("{:?}", e),
```

Comment:
@cjpatton any recommendation here to avoid panic?

Reply:
I would probably just propagate the DapError here. I think it's fine to panic, as long as the only way to panic is an RNG failure?

Reply:
it is possible that kem_id is invalid, so better to propagate error.